src/ui/components/pages/swap/hooks/useSwappableAssetTickersByVendor.ts

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from 'react';
+import { SwapVendor } from 'swap/constants';
+import { useAppSelector } from 'ui/store';
+import { assetIds } from 'assets/constants';
+import { NetworkName } from 'networks/types';
+
+export type SwappableAssetIdRecord = Record<NetworkName.Mainnet, string[]>;
+
+export const useSwappableAssetTickersByVendor = (): {
```

Comment:
Why we need here arrow function?

Reply:
Why not? Any significant difference?

Reply:
It's kind of a design convention.
We don't use arrow functions outside the scope. 
It also makes more sense when you see a declaration of `function` instead of `const`.